### PR TITLE
Added views to infoblox dnsapi script

### PR DESCRIPTION
**Pull request by hajhatten**
Views are used to setup "split horizon" dns.
The script only used the default view, and wasn't compatible with our setup.

This change lets you setup a Infoblox_View environment variable. But defaults to the one used before: "default", if not set.

### CODE REVIEW

**Comment by hemskgren** [comment]
Great, it was needed.

**Comment by Neilpang** [comment]
@hajhatten 
The `Infoblox_View` variable should be saved if it's not default value, just like `Infoblox_Creds`

**Comment by hajhatten** [comment]
Right, I wasn't sure that was needed. Fixed now.

File: dnsapi/dns_infoblox.sh [hunk 1]
```diff
@@ -94,4 +99,4 @@ dns_infoblox_rm() {
   fi
 }
 
-####################  Private functions below ##################################
+####################  Private functions below ##################################
```
Comment (Neilpang) [inline dnsapi/dns_infoblox.sh]: Please add new line at the end of the file.

**Pull request merged**